2_observations.yml

```diff
@@ -74,8 +84,18 @@ targets:
   2_observations/out/obs_flow_drb.rds.ind:
     command: munge_flow(
       out_ind = target_name,
-      dat_ind = '2_observations/out/drb_discharge_daily_dv.csv.ind',
+      dat_ind = '2_observations/in/daily_flow.rds.ind',
       sites_ind = '2_observations/out/drb_filtered_sites.rds.ind')
 
   2_observations/out/drb_temp_summary.csv:
-    command: summarize_temp(in_ind = '2_observations/out/obs_temp_drb.rds.ind', out_file = target_name)
+    command: summarize_dat(in_ind = '2_observations/out/obs_temp_drb.rds.ind', out_file = target_name)
+
+  2_observations/out/drb_flow_summary.csv:
+    command: summarize_dat(in_ind = '2_observations/out/obs_flow_drb.rds.ind', out_file = target_name)
+
+  # reservoir release data
+  2_observations/out/reservoir_releases.csv.ind:
+    command: clean_release_dat(
+      in_ind = '2_observations/in/PCN_Spill_Release.xlsx.ind',
+      out_ind = target_name,
+      mgd_to_cfs = I(1.54723))
```

Comment:
it might be better to have these as `cms`. That is what the observations are and the predictions from the PGDL and PRMS model

Reply:
that conversion factor would be 0.052616782407407 cms/mgd

Reply:
I think 1,000,000 gallons/day translates to 0.0438 cms